test/extended/cli/explain.go

```diff
@@ -579,6 +602,9 @@ var _ = g.Describe("[sig-cli] oc explain", func() {
 		if *controlPlaneTopology == configv1.ExternalTopologyMode {
 			crdTypesTest = baseCRDTypes
 		}
+		if !doesCRDTypesExist(oc, crdTypesTest) {
```

Comment:
Instead of skipping the test entirely, let's look for MicroShift and set crdTypesTest to the list we expect to see so we can test that explain works with them.